config/openiotsdk/CMakeLists.txt

```diff
@@ -149,7 +149,7 @@ else()
     set(CHIP_PROJECT_CONFIG "")
 endif()
 
-if (${CMAKE_BUILD_TYPE} STREQUAL "Debug")
+if (NOT ${CMAKE_BUILD_TYPE} STREQUAL "Release")
```

Comment:
Why change the test ? There's 4 default config: `Debug, Release, RelWithDebInfo and MinSizeRel`. 

The only debugging one is `Debug`.

Reply:
Remnants of tests, to be turned over.